src/psycopmlutils/loaders/raw/load_coercion.py

```diff
@@ -87,42 +92,209 @@ def _concatenate_coercion(
             .reset_index(drop=True)
         )
 
-    @data_loaders.register("coercion_farlighed")
-    def coercion_farlighed(n: Optional[int] = None) -> pd.DataFrame:
+    ### REASON (begrundtekst_sei) ###
+
+    @data_loaders.register("farlighed")
+    def farlighed(n: Optional[int] = None) -> pd.DataFrame:
 
-        coercion_types_lists = [
+        coercion_types_list = [
             {
-                "coercion_type": "Bælte",
                 "reason_for_coercion": "Farlighed",
             },
+            {
+                "reason_for_coercion": "På grund af farlighed",
+            },
+        ]
+
+        return LoadCoercion._concatenate_coercion(
+            coercion_types_list=coercion_types_list,
+            n=n,
+        )
+
+    @data_loaders.register("urolig_tilstand")
+    def urolig_tilstand(n: Optional[int] = None) -> pd.DataFrame:
+
+        return LoadCoercion.coercion_duration(
+            reason_for_coercion="Urolig tilstand",
+            n=n,
+        )
+
+    @data_loaders.register("anden_begrundelse")
+    def anden_begrundelse(n: Optional[int] = None) -> pd.DataFrame:
+
+        return LoadCoercion.coercion_duration(
+            reason_for_coercion="Anden begrundelse",
+            n=n,
+        )
+
+    @data_loaders.register("af helbredsmæssige grunde")
+    def af_helbredsmæssige_grunde(n: Optional[int] = None) -> pd.DataFrame:
+
+        return LoadCoercion.coercion_duration(
+            reason_for_coercion=" Af helbredsmæssige grunde",
+            n=n,
+        )
+
+    @data_loaders.register("nærliggende eller væsentlig fare for patienten eller andre")
+    def nærliggende_fare(n: Optional[int] = None) -> pd.DataFrame:
+
+        return LoadCoercion.coercion_duration(
+            reason_for_coercion="Nærliggende eller væsentlig fare for patienten eller andre  ",
```

Comment:
I imagine the spaces at the end of the `reson_for_coersion=`  string will be problemmatic.